From 6c9e84a021b24d98314e44c1063712596752e1aa Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@debian.org>
Date: Fri, 19 Jul 2019 14:49:19 +0000
Subject: [PATCH 1/3] Disable secctx in the default cachefilesd.conf (Closes:
 #909523).

This configuration line assumes that:

 - either there is no active LSM, which is wrong on Buster
   where AppArmor is enabled by default;

 - or SELinux is the active LSM, which is a rare configuration on Debian.

When this assumption is wrong, i.e. in most cases on current Debian
Buster/testing/sid, cachefilesd fails to start if this configuration
line is enabled.
---
 cachefilesd.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cachefilesd.conf b/cachefilesd.conf
index 6905281..bf14950 100644
--- a/cachefilesd.conf
+++ b/cachefilesd.conf
@@ -21,4 +21,4 @@ fstop 3%
 
 # Assuming you're using SELinux with the default security policy included in
 # this package
-secctx system_u:system_r:cachefiles_kernel_t:s0
+# secctx system_u:system_r:cachefiles_kernel_t:s0
-- 
2.22.0

